### PR TITLE
EOS:20886: No fault alerts seen in Kafka for CPU, diskspace and host_…

### PR DESCRIPTION
**Pull request by gargpalak048**
…memory usage

## Problem Statement:

No fault alerts seen in Kafka for CPU, diskspace and host_memory usage

## Solution Design:
Reduced the wait threshold time for cpu, memory sensors so the alerts can be received in that time limit.

## Coding:

* [ ] Did you follow coding standard and verified it with flake8? 
yes
* [ ] Did your address all Codacy issues?
yes

## Testing:

* [ ] Unittest updated and all unittests are passing?
No test-case updation required for a fix.

* [ ] Sanity/Self tests are updated (if applicable)?
no

* [ ] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [ ] Did you add Jira number to the PR?
yes

* [ ] DCO and cla-signed?
yes

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml [hunk 1]
```diff
@@ -71,8 +71,8 @@ LOGGINGMSGHANDLER:
 
 NODEDATAMSGHANDLER:
    transmit_interval: 10
-   high_cpu_usage_wait_threshold: 30
-   high_memory_usage_wait_threshold: 30
+   high_cpu_usage_wait_threshold: 10
```
Comment (thavanathan) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml:74]: I'm assuming our test code will wait for 10 seconds before checking CPU usage again? Is that right?

**Review by jjxsg — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml (hunk 1 above)
Comment (gargpalak048) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml:74]: @thavanathan the cpu/memory sensors raises alert only if usage remains over the threshold for the defined time period.
Earlier these wait threshold frequency was 30sec which was high and beyond test limit,
So I reduced the time accordingly and now alerts can be seen in this time period

**Review by thavanathan — APPROVED** [review]

File: low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml [hunk 2]
```diff
@@ -71,8 +71,8 @@ LOGGINGMSGHANDLER:
...
+   high_memory_usage_wait_threshold: 20
```
Comment (sumedhak27) [inline low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml:75]: @gargpalak048, better way is to keep tests unaffected by actual sspl config, 
setting test values before test starts and resetting them at the end is better I think.

Can add the following in `setup/sspl_test.py` if everyone agrees 
```
            cpu_usage_alert_wait = Conf.get(SSPL_CONFIG_INDEX,
                "NODEDATAMSGHANDLER>high_cpu_usage_wait_threshold")

            memory_usage_alert_wait = Conf.get(SSPL_CONFIG_INDEX,
                "NODEDATAMSGHANDLER>high_memory_usage_wait_threshold")

            cpu_usage_alert_wait_new = memory_usage_alert_wait_new = 0

            Conf.set(SSPL_CONFIG_INDEX,
                     "NODEDATAMSGHANDLER>high_cpu_usage_wait_threshold",
                     cpu_usage_alert_wait_new)

            Conf.set(SSPL_CONFIG_INDEX,
                     "NODEDATAMSGHANDLER>high_memory_usage_wait_threshold",
                     memory_usage_alert_wait_new)

            Conf.save(SSPL_CONFIG_INDEX)

#### ### run_qa_test.sh call   ######################            

            Conf.set(SSPL_CONFIG_INDEX,
                     "NODEDATAMSGHANDLER>high_cpu_usage_wait_threshold",
                     cpu_usage_alert_wait)
            Conf.set(SSPL_CONFIG_INDEX,
                     "NODEDATAMSGHANDLER>high_memory_usage_wait_threshold",
                     memory_usage_alert_wait)
```

Comment (gargpalak048) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml:75]: @sumedhak27, I believe 30s is a high time period to wait for sensors to raise a alert
Also, why to set the wait value as 0 while running test-cases?
shouldn't we test the sensors with actual config? 
> cpu_usage_alert_wait_new = memory_usage_alert_wait_new = 0

@jjxsg @cdeshmukh could you also share your opinions on this

Comment (cdeshmukh) [reply low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml:75]: >better way is to keep tests unaffected by actual sspl config, setting test values before test starts and resetting them at the end is better I think.

@sumedhak27 is right, this is what i meant in scrum today. and not to change original sspl config itself.

Just before sanity tests, set these 
> high_cpu_usage_wait_threshold: 10
 high_memory_usage_wait_threshold: 20

restart sspl, run tests. revert config, restart sspl. This is what we do for current tests too

**Review by cdeshmukh — APPROVED** [review]

**Comment by cdeshmukh** [comment]
test this please

**Pull request merged**